packages/cli-kit/src/node/cli.ts

```diff
@@ -78,21 +78,28 @@ export async function runCreateCLI(options: RunCLIOptions) {
 
 const reportError = async (errorToReport: Error): Promise<Error> => {
   await reportEvent({errorMessage: errorToReport.message})
+  if (settings.debug || !shouldReportError(errorToReport)) return errorToReport
 
-  if (!settings.debug && shouldReportError(errorToReport)) {
-    let mappedError: Error
-    // eslint-disable-next-line no-prototype-builtins
-    if (Object.prototype.isPrototypeOf(errorToReport)) {
-      const mappedError = Object.assign(Object.create(errorToReport), {})
-      if (mappedError.stack) mappedError.stack = mappedError.stack.replace(new RegExp('file:///', 'g'), '/')
-    } else {
-      mappedError = errorToReport
+  let mappedError: Error
+
+  // eslint-disable-next-line no-prototype-builtins
+  if (Error.prototype.isPrototypeOf(errorToReport)) {
+    mappedError = new Error(errorToReport.message)
+    if (errorToReport.stack) {
+      // For mac/linux, remove `file:///` from stacktrace
+      // For windows, remove `file:///C:/` from stacktrace
+      mappedError.stack = errorToReport.stack.replace(new RegExp('file:///([a-zA-Z]:/)?', 'g'), '/')
```

Comment:
What about removing everything before `node_modules`?

- `file:///C:/Users/isaac/test-bugsnag/node_modules/@shopify/cli/src/index.ts:52` => `@shopify/cli/src/index.ts:52`
- `file:///home/gonzalo/projects/my-hydrogen-app/node_modules/swiper/swiper.cjs.js:14` => `swiper/swiper.cjs.js:14`

Reply:
Umm, we could do it, bugsnag works as it is right now, would it be useful to apply this? 🤔

Reply:
That part can contain sensitive information and we don't need it...

Reply:
I can definitely remove it from the stack, but bugsnag will still report the full path... (don't know how they do it)

Reply:
Updated the regular expression, you can see how it behaves here: https://regex101.com/r/1lvUqX/1